rust/src/db.rs

```diff
@@ -353,9 +384,11 @@ mod tests {
             .set_default()
     }
 
-    async fn ensure_init_fresh_db() -> Result<()> {
+    // Prepare DB for the test. Each test should have DB with unique name to
+    // avoid race conditions.
```

Comment:
🙃 What about a random name? It's harder to make a mistake then and easier to use.

Reply:
if there were more than 2 tests, then yes :P didn't want to invest more time than absolutely needed at this point ;D